sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -3765,6 +3775,8 @@ class AzureBlobFileListSource:
     """Azure Blob Storage container URL."""
     file_list: str
     """Path to a JSONL file within the container specifying a subset of documents for training."""
+    _kind: Literal["azureBlobFileList"]
```

Comment:
why make it "private"?

Reply:
I didnt want to expose extra stuff, probably just being cautious, but I'd rather wait until the next preview when the service  adds this in before exposing more stuff officially.

Reply:
If we intend to keep it private, we should remove it from the class vars

Reply:
Done!